ops/swagger_to_openapi.sh

```diff
@@ -8,4 +8,8 @@ set -o pipefail
 java -jar -DswaggerUrl=swagger.yaml /swagger-converter/jetty-runner.jar /swagger-converter/server.war &
 sleep 30
 curl -X POST -d "@swaggerui/swagger.json" localhost:8080/api/convert -H "Content-Type: application/json" -o temp.json
```

Comment:
I wonder if we can inspect a response code to determine failures? 

I understand what line 15 is doing, but we could get some failures if we had some slight name changes.

Reply:
The problem which motivated this change occurred as a result of a successful request (`200`), but an invalid `openapi.json` file returned.  If we `grep` for the proper text to exist in the returned file (if a file is returned), do we cover our bases?

Reply:
Ah I see. Didn't know that we were receiving a 200 OK from the converter.